lib/screens/profile/profile.dart

```diff
@@ -0,0 +1,63 @@
+import 'package:ecellapp/screens/profile/cubit/profile_cubit.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+
+class ProfileScreen extends StatefulWidget {
+  @override
+  _ProfileScreenState createState() => _ProfileScreenState();
+}
+
+class _ProfileScreenState extends State<ProfileScreen> {
+  Map<String, dynamic> json;
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: BlocConsumer<ProfileCubit, ProfileState>(
+        listener: (context, state) {
+          if (state is ProfileError) {
+            Scaffold.of(context).showSnackBar(
+              SnackBar(content: Text(state.message)),
+            );
+          }
+        },
+        builder: (context, state) {
+          if (state is ProfileSuccess) {
+            json = state.json;
+            return _buildSuccess(context);
+          } else {
```

Comment:
this will call the _profile when state is ProfileError too, fix it

Reply:
done